<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="RegressionSuite" duration-ms="7134" started-at="2022-02-12T21:48:26Z" finished-at="2022-02-12T21:48:33Z">
    <groups>
    </groups>
    <test name="MyAccount" duration-ms="1227" started-at="2022-02-12T21:48:26Z" finished-at="2022-02-12T21:48:27Z">
      <class name="YourStoreTestCases.MyAccountTestCase">
        <test-method status="PASS" signature="myAccount()[pri:0, instance:YourStoreTestCases.MyAccountTestCase@617faa95]" name="myAccount" duration-ms="816" started-at="2022-02-12T16:48:26Z" finished-at="2022-02-12T16:48:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myAccount -->
      </class> <!-- YourStoreTestCases.MyAccountTestCase -->
    </test> <!-- MyAccount -->
    <test name="Register" duration-ms="4112" started-at="2022-02-12T21:48:27Z" finished-at="2022-02-12T21:48:31Z">
      <class name="YourStoreTestCases.RegisterTestCase">
        <test-method status="FAIL" signature="Register()[pri:0, instance:YourStoreTestCases.RegisterTestCase@6c3708b3]" name="Register" duration-ms="3892" started-at="2022-02-12T16:48:27Z" finished-at="2022-02-12T16:48:31Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Your Account Has Been Created!] but found [Register Account]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Your Account Has Been Created!] but found [Register Account]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at YourStoreTestCases.RegisterTestCase.Register(RegisterTestCase.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Register -->
      </class> <!-- YourStoreTestCases.RegisterTestCase -->
    </test> <!-- Register -->
    <test name="Login" duration-ms="1768" started-at="2022-02-12T21:48:31Z" finished-at="2022-02-12T21:48:33Z">
      <class name="YourStoreTestCases.LoginTest">
        <test-method status="PASS" signature="init()[pri:0, instance:YourStoreTestCases.LoginTest@6ddf90b0]" name="init" is-config="true" duration-ms="6165" started-at="2022-02-12T16:48:20Z" finished-at="2022-02-12T16:48:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="login()[pri:0, instance:YourStoreTestCases.LoginTest@6ddf90b0]" name="login" duration-ms="1718" started-at="2022-02-12T16:48:31Z" finished-at="2022-02-12T16:48:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:YourStoreTestCases.LoginTest@6ddf90b0]" name="tearDown" is-config="true" duration-ms="99" started-at="2022-02-12T16:48:33Z" finished-at="2022-02-12T16:48:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- YourStoreTestCases.LoginTest -->
    </test> <!-- Login -->
  </suite> <!-- RegressionSuite -->
</testng-results>
